module/proto/gravity/v1/genesis.proto

```diff
@@ -112,12 +113,12 @@ message Params {
 message GenesisState {
   Params params = 1;
   uint64 last_observed_event_nonce = 2;
-  repeated google.protobuf.Any outgoing_txs = 3;
+  repeated google.protobuf.Any outgoing_txes = 3;
```

Comment:
mostly to solve lint issue. 

modifying those field are not likely to be an issue as they are only used in the genesis state